PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/CustomTooltipOnHover.java

```diff
@@ -0,0 +1,150 @@
+package com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool;
+
+import com.intellij.codeInspection.LocalQuickFix;
+import com.intellij.codeInspection.ProblemDescriptor;
+import com.intellij.ide.BrowserUtil;
+import com.intellij.openapi.editor.Editor;
+import com.intellij.openapi.editor.colors.EditorColorsManager;
+import com.intellij.openapi.editor.colors.EditorFontType;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.ui.popup.Balloon;
+import com.intellij.openapi.ui.popup.JBPopupFactory;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiFile;
+import com.intellij.ui.JBColor;
+import com.intellij.ui.awt.RelativePoint;
+import com.intellij.ui.components.JBPanel;
+import com.intellij.util.ui.JBUI;
+import com.intellij.openapi.editor.EditorFactory;
+import org.jetbrains.annotations.NotNull;
+
+import javax.swing.JEditorPane;
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.Insets;
+import javax.swing.event.HyperlinkEvent;
+import javax.swing.text.html.HTMLEditorKit;
+import javax.swing.text.html.StyleSheet;
+
+/**
+ * This class is used to create a custom quick fix that shows a tooltip with a recommendation text and a link to the Azure SDK for Java documentation.
+ */
+class CustomTooltipOnHover implements LocalQuickFix {
+
+    private final String recommendationText;
+    private final String linkUrl;
+
+    /**
+     * Constructor for CustomTooltipOnHover.
+     *
+     * @param recommendationText - the recommendation text to be shown in the tooltip
+     * @param linkUrl            - the URL to be opened when the user clicks on the link in the tooltip
+     */
+    CustomTooltipOnHover(String recommendationText, String linkUrl) {
+        this.recommendationText = recommendationText;
+        this.linkUrl = linkUrl;
+    }
+
+    /**
+     * This method is used to get the name of the quick fix.
+     *
+     * @return The name of the quick fix.
+     */
+    @Override
+    public @NotNull String getName() {
```

Comment:
Will this be updated in the future to  get "the respective quick fix action" lookup?

Reply:
this is the "show details" you click on to bring up on the additional info. When I'm working on the action stretch goal I'll find an alternative so the hover popup isn't too busy